README.md

```diff
@@ -50,3 +50,61 @@ Example:
 ```
 
 [Yarn docs reference for this](https://yarnpkg.com/getting-started/qa#how-to-share-scripts-between-workspaces)
+
+
+### Versioning / Publishing Packages
+
+If you are contributing a user-facing or noteworthy change to a pie-monorepo package that should be added to the changelog, you should include a changeset with your PR.
+
+Changesets are only required for Major, Minor, and patch changes that have an effect on consumers. Changes that don't affect consumers do not need to be versioned. These include: linting, testing, CI.
+
+To add a changeset, run this script locally in the root of the project:
+
+```bash
+yarn changeset
+```
+
+Follow the prompts to select which package(s) are affected by your change, and whether the change is a major, minor or patch change. This will create a file in the `.changesets` directory of the repo. This change should be committed and included with your PR.
+
+Considerations:
+
+- You can use markdown in your changeset to include code examples, headings, and more. However, **please use plain text for the first line of your changeset**. The formatting of the GitHub release notes does not support headings as the first line of the changeset.
+- When selecting packages for the changesets, only select packages which are published.
+
+
+#### Stable Versions - 'latest' tag
+
+If your change is intended to be released under the `latest` tag on npm, you must follow this workflow:
+
+- Create a branch with your changes.
+- When you create your PR, target the `main` branch.
+
+Upon merging to `main`, the CHANGELOG.md, package.json version bump, and npm publish will execute automatically by GitHub Actions.
```

Comment:
Just a quick one – I thought once changes were merged to `main`, changesets would open a new PR where the updates would need to be merged from? (so multiple changes would get released together then)

Reply:
Ah yes, will amend this!